### ISSUE TITLE
Fix request: false positive for HashCodeOnlyCallsSuper -- not when equals also calls only super

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#483]
When equals also only calls super.equals no violation.

```java
    @Override
    public boolean equals(Object o) {
        return super.equals(o);
    }

    @Override
    public int hashCode() {
        return super.hashCode();  // <---- false positive
    }
```

Note: if there are object fields not used in equals/hashCode... it might be an issue or not? Other rule? It is not always clear if fields should be logically part of equals/hashCode, it depends.

**Comment by jborgers** [comment]
There is another rule for equals in value objects, some conditions apply, maybe at least 2 fields (not sure) and corresponding getter(s). Every non-dependent field should take part in the equals.
